vhost-device-sound/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 ### Added
 
+- [[#806]](https://github.com/rust-vmm/vhost-device/pull/806) Add controls field in VirtioSoundConfig
```

Comment:
Is not the CHANGELOG updated only when releasing a new version? I may be wrong.

Reply:
IIRC, the idea was to add changelogs incrementally to reduce the stress of finding all the commits related to the crate before a release.